dpctl/tensor/libtensor/include/kernels/copy_as_contiguous.hpp

```diff
@@ -150,9 +160,11 @@ sycl::event submit_c_contiguous_copy(sycl::queue &exec_q,
     const std::size_t lws =
         ((preferred_lws + max_sg_size - 1) / max_sg_size) * max_sg_size;
 
-    constexpr std::uint32_t nelems_per_wi = n_vecs * vec_sz;
-    size_t n_groups =
-        (nelems + nelems_per_wi * lws - 1) / (nelems_per_wi * lws);
+    constexpr std::uint8_t nelems_per_wi = n_vecs * vec_sz;
```

Comment:
If we're going to use `uint8_t` here, maybe a note should be made at the top of the functor or in the impl function that we assume `n_vecs * vec_sz` fits in the type.

Reply:
I added static assert to verify that.